.rhcicd/pr_check_fp.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -exv
+
+# Clowder config
+export APP_NAME="policies"
+export COMPONENT_NAME="policies-ui-backend"
+export IMAGE="quay.io/cloudservices/iqe-tests:policies"
+export DEPLOY_TIMEOUT="600"
+
+# IQE plugin config
+export IQE_PLUGINS="floorist"
+export IQE_MARKER_EXPRESSION="floorist_smoke"
+export IQE_FILTER_EXPRESSION=""
+export IQE_CJI_TIMEOUT="30m"
+
+# Bonfire init
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
+# Build the image and push to Quay
+export DOCKERFILE=src/main/docker/Dockerfile.jvm
+source $CICD_ROOT/build.sh
+
+# Deploy on ephemeral
+export COMPONENTS_W_RESOURCES="policies-engine"
+source $CICD_ROOT/deploy_ephemeral_env.sh
+
+# Run smoke tests with ClowdJobInvocation
+source $CICD_ROOT/cji_smoke_test.sh
+
+# Until test results produce a junit XML file, create a dummy result file so Jenkins will pass
+mkdir -p $WORKSPACE/artifacts
+cat << EOF > ${WORKSPACE}/artifacts/junit-dummy.xml
+<testsuite tests="1">
+    <testcase classname="dummy" name="dummytest"/>
```

Comment:
For next improvment, it would be good to provide a proper test report.

Reply:
I'll be happy to look into this more. However, at first glance it appears that this is related to the unit tests. We get results for all of the iqe-tests. These show up as (root) in the output.